Procfile

```diff
@@ -1 +1,11 @@
 web: npm start
+<<<<<<< HEAD
+=======
+<<<<<<< HEAD
+web: npm start
+=======
+web: npm start
+>>>>>>> 3ede931... crud route tests
+>>>>>>> added Get Post Delete Put for routes
+=======
+>>>>>>> fixed
```

Comment:
Please remove those Unnecessary lines in the file and remember to squash the commits to one commit